### ISSUE TITLE
Track listing won't load since 0.7.3

### ISSUE THREAD

**Issue by westingham** [jeffvli/feishin#765]
### Expected Behavior

Clicking on the Tracks category should display tracks and allow you to play those tracks.
![tracks](https://github.com/user-attachments/assets/45761672-5c97-4b2d-9ce2-7952df6ef59f)

### Current Behavior

Clicking on the Tracks category doesn't display any tracks. Trying to switch the sorting shows a spinning circle but that's it.
![notracks](https://github.com/user-attachments/assets/e96e518c-96b2-4d02-a2fa-591e439dd73f)

### Steps to Reproduce

Install any version of feishin after 0.7.3
Click on the Tracks category
Get a blank track listing
Install feishin 0.7.3
Click on the Tracks category
See tracks again

### Context

I'd like to use newer versions of feishin but I'm currently stuck at 0.7.3 due to this bug.

It should be noted that all of the other categories (Home, Albums, Artists, etc.) work as expected and I can play tracks there. It's just the Tracks category that doesn't work.

### Application version

0.8.0+

### Operating System and version

Windows 10

### Server and Version

Jellyfin 10.9.11

**Comment by jeffvli** [comment]
You're going to have to check the devtools console to see if you're getting any errors when navigating to the track list page.

Testing Jellyfin v10.9.11 works fine for me.

**Comment by westingham** [comment]
No error received when navigating to the track list page.
When I try to play a song from the list, I get an error.
![error](https://github.com/user-attachments/assets/eeb31fe5-5bf7-4eb4-8fc0-e781066fd9d6)

**Comment by jeffvli** [comment]
Does the network tab show that fetch requests are coming through if you click the refresh button?

If so, click on the individual network requests and check if it's returning a valid result.

**Comment by westingham** [comment]
Clicking the Refresh button doesn't generate anything in the Network tab.
However, Feishin is definitely trying to do something on the track list page.
![network1](https://github.com/user-attachments/assets/616efef8-1193-4d1e-914e-30f37b6380e5)

**Comment by westingham** [comment]
Maybe this will help. I'm including a screenshot of the network tab when I open the track list page in 0.7.3 and when I do the same in 0.10.1.
While all the responses are showing a 200 result, which I understand to be good, it seems like it's still trying to find something. It just does as bunch of those same requests and then stops.
![073](https://github.com/user-attachments/assets/e677da77-a97d-4019-b958-d7659eebd917)
![0101](https://github.com/user-attachments/assets/2dc47134-4289-4c67-bcd7-5f1897bbd441)

**Comment by jeffvli** [comment]
That's still not enough information to fully debug the issue. Is 0.10.0 the first version where it doesn't work? What about the versions inbetween?

Could you copy each of the network request query params so we can see what it's actually fetching as well?

**Comment by westingham** [comment]
The issue first appeared in 0.8.0 and has continued since then.

Here is the information about the network request queries that you asked for. The version used is 0.10.1. Let me know if you want a different version.

[items.txt](https://github.com/user-attachments/files/17181315/items.txt)
[items-fields.txt](https://github.com/user-attachments/files/17181316/items-fields.txt)

**Comment by jeffvli** [comment]
The network requests look fine to me (although I would also like to request one where you see `Limit` of 500 and its full response)

![image](https://github.com/user-attachments/assets/fdb8bc93-167f-4eca-8fc0-76bd679a1130)

--- 
Other things to try:

1. Try changing the folder filter to your primary music folder

![image](https://github.com/user-attachments/assets/445284cd-199e-4de7-a4e5-5fa4a41cab83)

2. Open the track list filters popup and try to set and reset all the filters (even if they're empty, just enter a value and press backspace until empty)

![image](https://github.com/user-attachments/assets/48c0ff11-a8d3-4dc7-8bc2-bf4e9f7c8c6d)

4. Try switching the list display type to the `Card` or `Poster` and see if those work

![image](https://github.com/user-attachments/assets/f210470a-9342-40bb-8353-b7326e05a8ed)

5. Completely reset your Feishin settings and config by going to `%appdata%\feishin\`  or `C:\Users\<your profile name>\AppData\Roaming\feishin` and removing or renaming the `Local Storage` folder and then restarting the app and reconfiguring.

**Comment by westingham** [comment]
Attached is the information from a request that has a payload value of 500.
[payload500.txt](https://github.com/user-attachments/files/17183266/payload500.txt)

1.) Changing the folder filter did not make anything different happen.

2.) Entering values, and then removing values, to the track list filters did not make anything different happen.

3.) Changing the display type to Card showed songs but only if the order is in Ascending view. I can scroll up and down through my entire track list and all the songs show up but as soon as I change the order to Descending view, then no songs will show up.
Changing the display type of Poster has the same effect as changing to Card.
Changing the display type back to Table will show the songs as long as the sort type is in Ascending.
Having said all that, I believe it's cached information of some sort. If I change list from Recently Added to Album or Artist or anything else, then no songs will show up regardless of whether I select Ascending or Descending.

4.) Deleting the Local Storage folder did not make anything different happen aside from what was mentioned in the above point.

**Comment by jeffvli** [comment]
Given the network request details, it should work... and I can't reproduce this nor have I ever seen anyone else have this issue.

I'm sorry, but there's not much more I can do to debug unless I can get temporary access to your server to test it myself. Is your Jellyfin instance public by any chance?

**Comment by westingham** [comment]
No, sorry, there's no external access to the server. 
Thank you for your help! I'll just continue to use 0.7.3 and try new versions as they come out.

**Comment by jeffvli** [comment]
Sounds good. Feel free to re-open this issue if you have any further updates.

**Comment by westingham** [comment]
For what it's worth, I tried installing 17.0 and this bug is no longer happening.

**Comment by TempestoGuy** [comment]
I had this bug too for a long time in any version above 0.7.3
I found out later it was because I had a song with an illegal character in the metadata and title (although not in the filename).
not sure if it was the only reason but everything worked in the new versions when i fixed it and rescanned.

other than the ones mentioned in this issue there is a bunch more I used to face in new versions that seem related
- the play random button would freeze/load tracks indefinitely.
- the play button on the tracks page would also show the notification and load indefinitely.
- playlists would also not show any tracks (just an empty loading page), apparently all the playlists i tested had that one song.
- when adding a track to a playlist with the `skip duplicates` option it would also load indefinitely and not add the track (worked fine without the option though) , since it probably had to read the playlist to check for duplicated entries.

all of these are fixed for me now
